apps/gallery/.storybook/preview.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { Controls, Description, Primary, Source, Stories, Subtitle, Title } from '@storybook/blocks'
 
+import '../css/index.css'
```

Comment:
Is it possible to set storybook to dark mode by default instead, thus avoiding its customisation (let's try to keep it as out of the box as possible so we don't spend too much time on it atm and can easily upgrade in the future in case they change classes)